tests/test_helpers/commands_helpers.ts

```diff
@@ -766,15 +767,18 @@ export function setSelection(
 
   if (zones.length !== 0) {
     const z1 = zones.splice(0, 1)[0];
-    model.selection.selectZone({ cell: { col: z1.left, row: z1.top }, zone: z1 });
+    model.selection.selectZone(
+      { cell: { col: z1.left, row: z1.top }, zone: z1 },
+      { unbounded: options.unbounded }
+    );
     for (const zone of zones) {
       model.selection.addCellToSelection(zone.left, zone.top);
       model.selection.setAnchorCorner(zone.right, zone.bottom);
     }
     model.selection.addCellToSelection(anchor.zone.left, anchor.zone.top);
     model.selection.setAnchorCorner(anchor.zone.right, anchor.zone.bottom);
   } else {
-    model.selection.selectZone(anchor);
+    model.selection.selectZone(anchor, { unbounded: options.unbounded });
```

Comment:
```suggestion
    model.selection.selectZone(anchor, { scrollIntoView: true, unbounded: options.unbounded });
```
selectZone has an implicit default option `{ scrollIntoView: true }` that will be erased by this overwrite